snuba/redis.py

```diff
@@ -11,15 +11,15 @@
 RedisClientType = Union[StrictRedis, StrictRedisCluster]
 
 
-class RetryingStrictRedisCluster(StrictRedisCluster):
+class RetryingStrictRedisCluster(StrictRedisCluster):  # type: ignore
```

Comment:
mh why this ?

Reply:
The error is `Class cannot subclass 'StrictRedisCluster' (has type 'Any')`. Not sure if there's an alternate solution to fix this.

Reply:
ok, Please add a comment explaining why we silence that.

Reply:
sure

Reply:
Happens with a couple of our libraries with missing type hints, not sure if we want to try to do this in mypy.ini in future